### PR TITLE
feat: add `color` property to `CCard` and `CCardDialog`

### PR DESCRIPTION
**Pull request by ItsPierre**

### CODE REVIEW

File: src/components/CCard.vue [hunk 1]
```diff
@@ -114,6 +115,11 @@ const props = withDefaults(
      * Whether the loading bar should be shown or not.
      */
     loading?: boolean;
+
+    /**
+     * Applies specified color to the card - supports utility colors (for example success or purple) or css color (for example success or purple) or css color (#033 or rgba(255, 0, 0, 0.5))
```
Comment (johannes-huther) [inline src/components/CCard.vue]: why are the examples the same?

File: src/components/CCardDialog.vue [hunk 2]
```diff
@@ -80,6 +81,11 @@ withDefaults(
      * Whether the loading bar should be shown or not.
      */
     loading?: boolean;
+
+    /**
+     * Applies specified color to the card - supports utility colors (for example success or purple) or css color (for example success or purple) or css color (#033 or rgba(255, 0, 0, 0.5))
```
Comment (johannes-huther) [inline src/components/CCardDialog.vue]: see above

**Review by johannes-huther — CHANGES_REQUESTED** [review]

**Comment by jojomatik** [comment]
https://github.com/Crystal-Creations-GbR/crystal-components/compare/0c803a95a8f5ef553940dfbc09b8f04e1f3e0ebd..ec36ab61e1f489be3dad62214ed8ff395aec196a

**Review by jojomatik — APPROVED** [review]

**Review by johannes-huther — APPROVED** [review]

**Comment by johannes-huther** [comment]
https://github.com/Crystal-Creations-GbR/crystal-components/compare/0c803a95a8f5ef553940dfbc09b8f04e1f3e0ebd..ec36ab61e1f489be3dad62214ed8ff395aec196a

**Pull request merged**